go/libraries/doltcore/tablediff_prolly/diff_schema.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tablediff_prolly
```

Comment:
I'm not sure I understand this new package, what's the motivation?

Reply:
I wanted to share functionality between the `dtables` and `sqle` package

Reply:
sqle already depends on dtables, so let me just move these functions into `dtables`.

Reply:
ignore all the above, dtables is a child of sqle...